DESCRIPTION

```diff
@@ -38,16 +39,13 @@ Imports:
     tools,
     vctrs,
     writexl,
-    stringr,
-    labeling,
-    showtext,
```

Comment:
How does it work now without importing `showtext` and snaphot tests passing? Didn't we have a problem with it?

Reply:
It is listed in Suggest now, so it works